man/org.freedesktop.portable1.xml

```diff
@@ -249,7 +249,7 @@ node /org/freedesktop/portable1 {
         <listitem><para>mkdir</para></listitem>
       </itemizedlist>
       Note that an image cannot be attached if a unit that it contains is already present
-      on the system.</para>
+      on the system. Note that this method operates synchronously.</para>
```

Comment:
"synchronous" means many things. Maybe say explicitly what you mean by that here, i.e. that all listed operations are complete, and maybe slow?